packages/flutter/lib/src/material/text_field.dart

```diff
@@ -388,17 +403,25 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     if (!needsCounter)
       return effectiveDecoration;
 
-    final String counterText = '${_effectiveController.value.text.runes.length}/${widget.maxLength}';
+    final int currentLines = _effectiveController.value.text.runes.length;
+    final String counterText = '$currentLines/${widget.maxLength}';
```

Comment:
If talkback always reads this out as a fraction can we provide a better default as default? My guess is that most devs will forget to customize this....